solutions/src/test/java/ru/sberbank/school/task11/MyStreamsTest.java

```diff
@@ -42,8 +42,8 @@ void filter() {
 
     @Test
     void transform() {
-        List<Integer> integerList = MyStreams.of("apple", "banana", "orange", "lemon", "strawberry", "lemon", "strawberry").transform(String::length).toList();
-        List<Integer> integerList1 = Stream.of("apple", "banana", "orange", "lemon", "strawberry", "lemon", "strawberry").map(String::length).collect(Collectors.toList());
+        List<Integer> integerList = MyStreams.of("apple", "banana", "orange", "lemon", "strawberry", "lemon", "strawberry").transform(String::length).sorted(Integer::compareTo).toList();
+        List<Integer> integerList1 = Stream.of("apple", "banana", "orange", "lemon", "strawberry", "lemon", "strawberry").map(String::length).sorted(Integer::compareTo).collect(Collectors.toList());
```

Comment:
Line is longer than 120 characters (found 198).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1943289833>*